src/components/queryBuilder/OtelVersionSelect.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import { SelectableValue } from '@grafana/data';
+import { InlineFormLabel, Select, Switch as GrafanaSwitch, useTheme } from '@grafana/ui';
+import { versions as allVersions } from 'otel';
+import selectors from 'labels';
+
+interface OtelVersionSelectProps {
+  enabled: boolean,
+  onEnabledChange: (enabled: boolean) => void,
+  selectedVersion: string,
+  onVersionChange: (version: string) => void,
+  defaultToLatest?: boolean,
+}
+
+export const OtelVersionSelect = (props: OtelVersionSelectProps) => {
+  const { enabled, onEnabledChange, selectedVersion, onVersionChange, defaultToLatest } = props;
```

Comment:
why do we need `defaultToLatest`?

Reply:
It's not really necessary, I think I had some ideas on how it might be configured in the datasource. Perhaps if they didn't select their preferred version in the datasource, we could simply default it to the latest version. In that case I would probably do something like `defaultToLatest={!isOtelVersionConfigured}`, but we could also simply make this the default behavior. I should remove this